mne/io/brainvision/brainvision.py

```diff
@@ -134,14 +147,18 @@ def __init__(self, vhdr_fname, montage=None,
                 elif self._order == 'C':  # vectorized, channels, in rows
                     raise NotImplementedError()
             else:
+                n_data_ch = info['nchan'] - int(stim_channel)
                 f.seek(0, os.SEEK_END)
                 n_samples = f.tell()
                 dtype_bytes = _fmt_byte_dict[fmt]
                 offsets = None
-                n_samples = n_samples // (dtype_bytes * (info['nchan'] - 1))
+                n_samples = n_samples // (dtype_bytes * n_data_ch)
```

Comment:
How is this better than before?

Reply:
Sometimes you need the -1 sometimes you don't